packages/material-ui/src/useMediaQuery/useMediaQuery.d.ts

```diff
@@ -16,4 +16,4 @@ export interface Options {
   ssrMatchMedia?: (query: string) => MuiMediaQueryList;
 }
 
-export default function useMediaQuery(query: string, options?: Options): boolean;
+export default function useMediaQuery(query: string | string[], options?: Options): boolean;
```

Comment:
This is a hidden feature we don't want to make public, at least, not yet: https://github.com/mui-org/material-ui/pull/15678#discussion_r291818253.

Reply:
Reverted